modules/fedimint-dummy-client/src/db.rs

```diff
@@ -17,6 +17,15 @@ pub enum DbKeyPrefix {
     // Used to verify that 0x50 key can be written to, which used to conflict with
     // `DatabaseVersionKeyV0`
     ClientName = 0x50,
+    /// Prefixes between 0xb0..=0xcf shall all be considered allocated for
```

Comment:
The constants are OK, but comments are stale/inconsistent/wrong.

Reply:
:facepalm: copying is hard